app/controllers/ausleihe_controller.rb

```diff
@@ -224,17 +209,17 @@ def archive(old_lend_out)
     archived.save!
   end
 
-  def has_mixed_content?(old_folder_instances)
-    lend_outs = old_folder_instances.map { |i| i.old_lend_out }
-    lent = lend_outs.reject { |l| l.nil? }
-
-    if lent.empty?
-      mixed_content = false
-    else
-      mixed_content = (lent.count != lend_outs.count)
-    end
-
-    mixed_content
+  def invalid_input_for_switch_message(folder_list, lent)
+    lent_as_strings = lent
+                          .map { |f, barcode, _| "#{barcode} (#{f})" }
+                          .join(', ')
+    all_as_strings = folder_list
+                         .map { |f, barcode, _| "#{barcode} (#{f})" }
+                         .join(', ')
+
+    message = ["#{Time.new}: Eingabe enthält gemischte Ordner. Entweder Ausleihen oder Zurücknehmen."]
+    message << "Ordner-Exemplare: #{all_as_strings}, davon verliehen: #{lent_as_strings}"
+    join = message.join
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Useless assignment to variable - `join`.](https://www.codacy.com/app/lippertsjan/klausurtool-ror/file/2382693691/issues/source?bid=3141297&fileBranchId=3226363#l222)